tests/avocado/test_nec-usb-xhci.py

```diff
@@ -19,6 +21,21 @@ class TestNecUsbXhci(test.QemuTest):
     def setUp(self):
         self.vm_dst = None
         usbdevice = os.path.join(self.workdir, 'usb.img')
+
+        self.image = vmimage.get('Fedora')
```

Comment:
Does this test only work for Fedora? If not then we should simply require image and not over-complicate things ...